music.js

```diff
@@ -1,17 +1,28 @@
-// Define the Artist constructor function here, with a 'name' property that can be set at instantiation,
-// and a 'songs' property that is an empty array to start
-function Artist() {
-
+function Artist(name) {
+  this.name = name;
+  this.songs = [];
 }
 
-// Define the Song constructor function here, with 'title' and 'artist' properties that can be set at instantiation,
-// and a 'playCount' property that is an empty array to start
-function Song() {
-
-  // When a new song is instantiated, you'll want to add that song to its artist's 'songs' array.
-  // You'll probably want to do that in here somewhere...
+function Song(title, artist) {
+  this.title = title;
+  this.artist = artist;
+  this.playCount = 0;
+  Artist.prototype.add = function(title) {
```

Comment:
You probably don't want to put this line of code inside of the `Song` constructor function. This is a fine way to create method to add a song, but you probably want this method defined right below the `Artist` constructor function instead.